docs/04-guides/on-demand/01-upload-video-asset/02-api.mdx

```diff
@@ -110,7 +112,7 @@ To upload the asset to the Livepeer network locally, you'll need to make a POST
 request to generate an upload URL.
```

Comment:
Let's change the "Upload Locally" heading to "Direct Upload" and add the following text beneath it explaining the pros and cons vs the Resumable Media Upload method.

"A direct upload uploads the whole video asset in one HTTP request, as opposed to the resumable upload protocol, which uploads the video asset in multiple requests. Doing a direct upload reduces the number of HTTP requests but increases the chance of failures (such as connection failures) that can happen with large uploads."